user/tests/integration/slo/connect_time/connect_time.cpp

```diff
@@ -55,6 +55,13 @@ inline auto& network() {
 #endif
 }
 
+bool isQuectelRadio(int radioType) {
+    return (radioType == DEV_QUECTEL_BG96 || radioType == DEV_QUECTEL_EG91_E ||
+            radioType == DEV_QUECTEL_EG91_NA || radioType == DEV_QUECTEL_EG91_EX ||
+            radioType == DEV_QUECTEL_BG95_M1 || radioType == DEV_QUECTEL_EG91_NAX ||
+            radioType == DEV_QUECTEL_BG77 || radioType == DEV_QUECTEL_BG95_MF);
+}
+
```

Comment:
Unclear which quectel radio types need this delay so adding 'em all just in case.

Reply:
You can check the vendor bit instead then. `PLATFORM_NCP_MANUFACTURER(radioType) == PLATFORM_NCP_MANUFACTURER_QUECTEL`